image/Dockerfile

```diff
@@ -41,9 +41,9 @@ COPY golem_blender_app/requirements.txt /tmp
 RUN pip install -r /tmp/requirements.txt
 
 # Use following lines to work with the unreleased task-api module
-# RUN apt-get install -y git
-# RUN pip uninstall -y golem-task-api
-# RUN pip install git+https://github.com/golemfactory/task-api@da1df90188ce033039b39a1b15b2a69dd9f2c77a#subdirectory=python
+RUN apt-get install -y git
+RUN pip uninstall -y golem-task-api
+RUN pip install git+https://github.com/golemfactory/task-api@049f0f4da7a9f1214d2a2d54d4c2c389c3b2d939#subdirectory=python
```

Comment:
Couldn't this be achieved by simply putting this line in `requirements.txt`?

Reply:
Yes, but then `requirements.txt` would invalidate Docker cache and reinstall all requirements from scratch. This way it only reinstalls `task-api` module which is significantly faster when you reiterate often.